operations/operations/company-processes/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines/documentation-style-guide.md

```diff
@@ -309,6 +401,10 @@ The link renders as: [Mattermost Manifesto](https://www.mattermost.org/manifesto
 
 ### Internal links to Mattermost documentation
 
+When creating a link to another document in the Mattermost documentation, create a link with a relative URL.
```

Comment:
This is a most excellent point and I'm glad you've included it and provided context as to why a relative link is preferred). At some point in the future, we'll want to clean up all of the absolute links by converting them to relative links.